server/jetstream_cluster_test.go

```diff
@@ -10674,6 +10674,23 @@ func TestJetStreamClusterStreamReplicaUpdateFunctionCheck(t *testing.T) {
 	}
 
 	checkSubsPending(t, sub, 3*numMsgs)
+
+	// Make sure cluster replicas are current.
+	for _, r := range si.Cluster.Replicas {
+		if !r.Current {
```

Comment:
Does that need to be protected with a "checkFor()"?

Reply:
Do you see this one flapping?

Reply:
No, I did not. Was just wondering if the state (Current and number msgs in check below) are guaranteed to be up-to-date at this place in the test. If not, then it may cause flapping on Travis.

Reply:
I could not make it fail after 50 runs with race and non-race but made the change anyway, good call by you.